app/services/pdf_service.rb

```diff
@@ -0,0 +1,36 @@
+class PdfService
+  GROVER_OPTIONS = {
+    format: 'A4',
+    margin: {
+      top: '2cm',
+      bottom: '2cm',
+      left: '1cm',
+      right: '1cm',
+    },
+    viewport: {
+      width: 2400,
+      height: 4800,
+    },
+    emulate_media: 'print',
+
+    launch_args: ['--font-render-hinting=medium', '--no-sandbox', '--force-renderer-accessibility'],
+  }.freeze
```

Comment:
could be extracted to an initializer but either way. [example from civil apply](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/config/initializers/grover.rb)

Reply:
Think I'm going to leave this as an option passed in for now, cos it makes debugging and tweaking easier - don't have to restart the local server to adjust the settings.